akka-http-core/src/main/java/akka/http/javadsl/model/HttpMessage.java

```diff
@@ -55,6 +60,16 @@
      */
     ResponseEntity entity();
 
+    /**
+     *  Drains entity stream of this message
+     */
+    DiscardedEntity discardEntityBytes(Materializer materializer);
+
+    interface DiscardedEntity {
+        Future<Done> future();
```

Comment:
Please document "This future completes successfully once the underlying entity stream has been successfully drained (and fails otherwise)."